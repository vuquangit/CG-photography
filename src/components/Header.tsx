import React from "react";
import Link from "next/link";
import Head from "next/head";

const linkStyle = {
  marginRight: 15,
};

const Header: React.FC = () => (
  <div>
    <Head>
      <title>NextJS + TypeScript Boilerplate</title>
      <meta name="viewport" content="initial-scale=1.0, width=device-width" />
    </Head>

    <Link href="/">
      <a style={linkStyle}>Home</a>
    </Link>

    <Link href="/about">
      <a style={linkStyle}>About</a>
    </Link>
  </div>
);

export default Header;
