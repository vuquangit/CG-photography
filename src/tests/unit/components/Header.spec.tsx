import * as React from "react";
import { render } from "@testing-library/react";

import Header from "@/components/Header";

describe("Header", () => {
  it("Test Header render", async () => {
    const { container, getByTestId } = render(<Header />);

    expect(container).toMatchSnapshot();
  });
});
