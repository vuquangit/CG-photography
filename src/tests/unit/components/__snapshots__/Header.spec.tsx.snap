// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Test Header render 1`] = `
<div>
  <div>
    <a
      href="/"
      style="margin-right: 15px;"
    >
      Home
    </a>
    <a
      href="/about"
      style="margin-right: 15px;"
    >
      About
    </a>
  </div>
</div>
`;
